chess2.cpp

```diff
@@ -0,0 +1,882 @@
+        {-20,-10,-10, -5, -5,-10,-10,-20},
+        {-10,  0,  0,  0,  0,  0,  0,-10},
+        {-10,  0,  5,  5,  5,  5,  0,-10},
+         {-5,  0,  5,  5,  5,  5,  0, -5},
+          {0,  0,  5,  5,  5,  5,  0, -5},
+        {-10,  5,  5,  5,  5,  5,  0,-10},
+        {-10,  0,  5,  0,  0,  0,  0,-10},
+        {-20,-10,-10, -5, -5,-10,-10,-20}
+    };
+    const int kingt[8][8] = {
+        {-30,-40,-40,-50,-50,-40,-40,-30},
+        {-30,-40,-40,-50,-50,-40,-40,-30},
+        {-30,-40,-40,-50,-50,-40,-40,-30},
+        {-30,-40,-40,-50,-50,-40,-40,-30},
+        {-20,-30,-30,-40,-40,-30,-30,-20},
+        {-10,-20,-20,-20,-20,-20,-20,-10},
+        {20, 20,  0,  0,  0,  0, 20, 20},
+        {20, 30, 10,  0,  0, 10, 30, 20 },
+    };
+    const int rookt[8][8] = {
+      {0,  0,  0,  0,  0,  0,  0,  0},
+      {5, 10, 10, 10, 10, 10, 10,  5},
+     {-5,  0,  0,  0,  0,  0,  0, -5},
+     {-5,  0,  0,  0,  0,  0,  0, -5},
+     {-5,  0,  0,  0,  0,  0,  0, -5},
+     {-5,  0,  0,  0,  0,  0,  0, -5},
+     {-5,  0,  0,  0,  0,  0,  0, -5},
+      {0,  0,  0,  5,  5,  0,  0,  0}
+    };
+    int score = 0;
+
+    for (int i = 0; i < 8; i++) {
+        for (int j = 0; j < 8; j++) {
+            if (!board[i][j]) continue;
+            if (board[i][j] == 1) {
+                score-=pawnt[i][j];
+                score -= pawn;
+                if (board[i-1][j] == 1) // double stacked pawn
+                    score-=20;
+            }
+            else if (board[i][j] == 2){
+                score-=bishopt[i][j];
+                score -= bishop;
+            }
+            else if (board[i][j] == 3){
+                score-=knightt[i][i];
+                score -= knight;
+            }
+            else if (board[i][j] == 5){
+                score-=rookt[i][j];
+                score -= rook;
+            }
+            else if (board[i][j] == 6){
+                score-=queent[i][j];
+                score -= queen;
+            }
+            else if (board[i][j] == 10){
+                score-=kingt[i][j];
+                score -= king;
+            }
+             if (board[i][j] == -1) {
+                score+=pawnt[7-i][7-j];
+                score+= pawn;
+            }
+            else if (board[i][j] == -2){
+                score+=bishopt[7-i][7-j];
+                score+= bishop;
+            }
+            else if (board[i][j] == -3){
+                score+=knightt[7-i][7-j];
+                score+= knight;
+            }
+            else if (board[i][j] == -5){
+                score+=rookt[7-i][7-j];
+                score+= rook;
+            }
+            else if (board[i][j] == -6){
+                score+=queent[7-i][7-j];
+                score+= queen;
+            }
+            else if (board[i][j] == -10){
+                score+=kingt[7-i][7-j];
+                score+= king;
+            }
+
+        }
+    }
+
+    return score;
+}
+
+int Chess2::miniMax(int depth, bool ismax, int alpha, int beta) {
```

Comment:
same as an above comment on boolean naming convention 
also has to be followed across the code
example use isMax not ismax